modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -410,9 +410,13 @@ namespace std
     };
 } // namespace std
 
-
 namespace cv {
+    class GAPI_EXPORTS_W_SIMPLE GKernelPackage;
+
 namespace gapi {
+    GAPI_EXPORTS GKernelPackage combine(const GKernelPackage  &lhs,
+                                        const GKernelPackage  &rhs);
```

Comment:
I think it is better to underline they are `cv::GKernelPackage` in this declaration (which is situated in `gapi` namespace, that's why it needs clarification)

Reply:
Applied